dstar_map/CellUpdate[] updates
